tools/scripts/ansible/requirements.yml

```diff
@@ -1,3 +1,6 @@
-sprylab.tomcat-ubuntu
+robertdebock.bootstrap
```

Comment:
it's not quite clear to me whether bootstrap/etc are needed here in addition to tomcat